proto/proxy/destination.proto

```diff
@@ -86,22 +86,9 @@ message WeightedAddr {
 // Which strategy should be used for verifying TLS.
 message TlsIdentity {
   oneof strategy {
-    K8sPodNamespace k8s_pod_namespace = 1;
     K8sPodIdentity k8s_pod_identity = 2;
   }
 
-  // Verify the certificate based on the Kubernetes pod name, and ensure
-  // that the pod is configured with the same Conduit control plane
-  // namespace as this proxy.
-  message K8sPodNamespace {
-    // The Kubernetes namespace of the pod's Conduit control plane.
-    string controller_ns = 1;
-    // The Kubernetes namespace that the pod is in.
-    string pod_ns = 2;
-    // The name of the pod.
-    string pod_name = 3;
-  }
-
```

Comment:
Not a problem, but this branch didn't include regenerated go protobuf bindings by running the `bin/protoc-go.sh` script. I can submit that as a separate PR if you'd like.

Reply:
> Not a problem, but this branch didn't include regenerated go protobuf bindings by running the bin/protoc-go.sh script. I can submit that as a separate PR if you'd like.

Thanks! It would be great if you could do it. I did try to do it but I don't have `jar` installed and I didn't know if it was going to be an ordeal after I saw that.

Reply:
Cool, I created #1223. There's a [fairly obscure reason](https://github.com/linkerd/linkerd/issues/1372) why we're using `jar` instead of `unzip`. Makes less sense in a go project though...